src/UKHO.ERPFacade.API/appsettings.json

```diff
@@ -38,7 +38,11 @@
         "BaseAddress": "",
         "SapServiceOperation": "Z_ADDS_MAT_INFO",
```

Comment:
This should be renamed now that we have more than one service operation?

Reply:
"SapConfiguration": {
        "SapEndpointBaseAddressForEncEvent": "",
        "SapServiceOperationForEncEvent": "Z_ADDS_MAT_INFO",
        "SapUsernameForEncEvent": "",
        "SapPasswordForEncEvent": "",
        "SapEndpointBaseAddressForRecordOfSale": "",
        "SapServiceOperationForRecordOfSale": "Z_ADDS_ROS",
        "SapUsernameForRecordOfSale": "",
        "SapPasswordForRecordOfSale": ""
    },


Hi John, Can we use above names for configuration?

Reply:
Updated the configuration names